website/content/demos/starknetid.mdx

```diff
@@ -86,3 +86,50 @@ function MyComponent() {
   );
 }
 ```
+
+## Lookup Stark profile
+
+Use the `useStarkProfile` hook to lookup the Starknet.ID associated data with a Starknet
+address.
+
+```tsx lookup-stark-profile.tsx
+import { useStarkProfile } from "@starknet-react/core";
+
+function MyComponent() {
+  const [address, setAddress] = useState<string>("");
```

Comment:
Can you add a default profile that's properly setup on testnet? So it will show data out of the box.